src/main/create-operator-json/core/operatorFactory.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+export default class Operator{
+
+    // must pass r6operator as first argumet
+    constructor(person, id, operation) {
+        this.id = id;
+        this.label = person.name;
+        this.side = person.role;
+        this.organization = person.unit;
+        this.operation = operation;
+        this.counters =[];
+    }
+
+    addCounterNode(personCountered, counterType, counterDescription){
+        this.counters.push({
+            type: counterType,
+            startNode: this.id,
+            endNode: personCountered,
+            properties: {
+                Description: counterDescription
+            }
+        });
+    }
+
+    getCounters() {
```

Comment:
Add a comment explaining what this method returns. Does it return operators this operator counters? Or is it operators this operator is countered by? Or both? Explain it in the comment.

Reply:
added comments